### ISSUE TITLE
transfer .csv file to sqlite wrong

### ISSUE THREAD

**Issue by lebinhe** [thombashi/sqlitebiter#33]
I have a xx.csv file like this:

smokey,Linux 3.0-ARCH,x86
12345678901,12345 1234567890123,123
12345678901,1234567890123456789,12345
11 bytes,19 bytes,5 byt
test line:,"Some ""comma, quote""",foo
skylight,Linux 3.0-ARCH,x86
polaris,Linux 3.0-ARCH,amd64
asgard,Windows 6.1.7600,amd64
galileo,Windows 6.2.8102,x86
kepler,Windows 6.2.8123,amd64
wrfbox,Windows 6.2.8133,amd64

I get the out.sqlite file when running "sqlitebiter file xx.csv" command.
But when i select from the table, i get the following columns:

12345678901|12345 1234567890123|123
12345678901|1234567890123456789|12345
11 bytes|19 bytes|5 byt
test line:|Some "comma, quote|foo
skylight|Linux 3.0-ARCH|x86
polaris|Linux 3.0-ARCH|amd64
asgard|Windows 6.1.7600|amd64
galileo|Windows 6.2.8102|x86
kepler|Windows 6.2.8123|amd64
wrfbox|Windows 6.2.8133|amd64

The quote has not transformed correctly at line:
    test line:,"Some ""comma, quote""",foo
missing right quote in sqlite table.

**Comment by thombashi** [comment]
@lebinhe
Thank you for your report.

I've fixed the problem at `sqlitebiter 0.8.1`.
Please upgrade `sqlitebiter` to `0.8.1`.

```
$ sqlitebiter --version
sqlitebiter, version 0.8.1
$ cat issue33.csv
smokey,Linux 3.0-ARCH,x86
12345678901,12345 1234567890123,123
12345678901,1234567890123456789,12345
11 bytes,19 bytes,5 byt
test line:,"Some ""comma, quote""",foo
skylight,Linux 3.0-ARCH,x86
polaris,Linux 3.0-ARCH,amd64
asgard,Windows 6.1.7600,amd64
galileo,Windows 6.2.8102,x86
kepler,Windows 6.2.8123,amd64
wrfbox,Windows 6.2.8133,amd64
$ sqlitebiter file issue33.csv
[INFO] sqlitebiter file: convert 'issue33.csv' to 'issue33' table
$ sqlite3 out.sqlite
sqlite> select * from issue33;
12345678901|12345 1234567890123|123
12345678901|1234567890123456789|12345
11 bytes|19 bytes|5 byt
test line:|Some "comma, quote"|foo
skylight|Linux 3.0-ARCH|x86
polaris|Linux 3.0-ARCH|amd64
asgard|Windows 6.1.7600|amd64
galileo|Windows 6.2.8102|x86
kepler|Windows 6.2.8123|amd64
wrfbox|Windows 6.2.8133|amd64
```

**Comment by thombashi** [comment]
@lebinhe
I'll close the issue.
Since I believe the problem fixed.
Feel free to reopen.
